Project.toml

```diff
@@ -1,6 +1,12 @@
 name = "ClimaComms"
 uuid = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
-authors = ["CliMA Contributors <clima-software@caltech.edu>"]
+authors = [
+    "Kiran Pamnany <clima-software@caltech.edu>",
+    "Simon Byrne <simonbyrne@caltech.edu>",
+    "Charles Kawczynski <clima-software@caltech.edu>",
```

Comment:
@charleskawczynski what email would you like me to use?

Reply:
My Caltech one is fine: `charliek@caltech.edu`

Reply:
```suggestion
    "Charles Kawczynski <charliek@caltech.edu>",
```